webapp/channels/src/plugins/mobile_channel_header_plug/mobile_channel_header_plug.tsx

```diff
@@ -104,16 +102,14 @@ class MobileChannelHeaderPlug extends React.PureComponent<Props> {
         }
 
         return (
-            <li className='flex-parent--center'>
```

Comment:
you may as well remove `flex-parent--center` from stylesheet since it is only used in this component